Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -371,4 +373,17 @@ class FileGenerator {
             p.print("]\n")
         }
     }
+
+    private func generateVersionCheck(printer p: inout CodePrinter) {
+        let v = Version.compatibilityVersion
+        p.print("// If the compiler emits an error on this type, it is because this file\n")
+        p.print("// was generated by a version of the `protoc` Swift plug-in that is\n")
+        p.print("// incompatible with the version of SwiftProtobuf to which you are linking.\n")
+        p.print("// Please ensure that your are building against the same version of the API\n")
+        p.print("// that was used to generate this file.\n")
+        p.print("fileprivate struct _GeneratedWithProtocGenSwiftVersion: SwiftProtobuf.ProtobufAPIVersionCheck {\n")
```

Comment:
Yeah, it's ok to use the module name within the same module. (In fact, it's required to access a global symbol within a scope where a more local name is shadowing it. That's not the case here, but it's harmless.)